### DISCUSSION TITLE
Feel free to ask questions here.

### DISCUSSION THREAD

**Discussion by jonaballe** [tensorflow/compression#76] [General]
If you have questions about how to use the library, or need help with implementing something, feel free to ask questions here.

You can create a new discussion topic by clicking on `Discussions`, and then the green `New discussion` button in the top right corner.

Please open issues only for bugs or feature requests.

Thanks!
Johannes.

**Comment by ankitlat** [comment]
Hi Johannes, 

I am eager to know is there a way to save encoded latent representations of the pre-trained models? Presently I can only save the compressed byte stream.

**Reply by jonaballe** [reply]
Please see my reply in #78.
